### PR TITLE
Add Author to Recipe

### PR DESCRIPTION
**Pull request by crowdedpoem**
With User login, now recipes have the author's username attached to their recipe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,53 @@
...
 
+### How to Install Prettier
 **Editor integration**
 https://prettier.io/docs/en/editors.html
 
 **Command-line**
 If you prefer to use command-line interface, type in the command
 <code> npm install --save-dev --save-exact prettier </code>
+
+NEED TO WALK THROUGH ON TEAM STYLEGUIDE AND HOW TO USE LOCALLY
```
Comment (Tpanyasr) [inline README.md:50]: I can add this once this PR is merged

**Review by Tpanyasr — APPROVED** [review]
Looks good overall

**Review by tylerboo — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (crowdedpoem) [reply README.md:50]: ok great, thanks

**Pull request merged**